pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -3234,7 +3231,7 @@ func TestFailTaskRun(t *testing.T) {
 				taskLister:        testAssets.Informers.Task.Lister(),
 				clusterTaskLister: testAssets.Informers.ClusterTask.Lister(),
 				resourceLister:    testAssets.Informers.PipelineResource.Lister(),
-				timeoutHandler:    nil, // Not used
+				snooze:            func(acc kmeta.Accessor, amnt time.Duration) {},
```

Comment:
Can you make this call t.Error so we're sure snooze isn't called in this case?

Reply:
also done.